docs/translations/translation-loading.md

```diff
@@ -0,0 +1,331 @@
+#: assets/js/blocks/product-tag/block.js:121
+#: assets/js/blocks/product-top-rated/block.js:36
+#: assets/js/blocks/products-by-attribute/block.js:46
+#: assets/js/blocks/single-product/edit/layout-editor.js:56
+msgid "Layout"
+msgstr ""
+
+[...]
+```
+
+See also https://developer.wordpress.org/plugins/internationalization/localization/#pot-portable-object-template-files.
+
+### PO (Portable Object) file
+
+PO stands for `Portable Object`, contains both the original and the translated strings and is based on the POT file. Similar to the POT file, it can also be created manually using [WP-CLI](https://wp-cli.org/) or [Poedit](https://poedit.net/). As mentioned before, this step is not necessary for the WooCommerce Blocks plugin, as it's hosted on WordPress.org. The PO file will be generated within 30 minutes after a new translation had been added via https://translate.wordpress.org/projects/wp-plugins/woo-gutenberg-products-block/.
+
+The PO file is human-readable and named `woo-gutenberg-products-block-{LANGUAGE-CODE}.po`, e.g. `woo-gutenberg-products-block-de_DE.po`. It can be found in the `/wp-content/languages/plugin` folder. The PO file looks like this:
+
+```po
+# Translation of Plugins - WooCommerce Blocks - Stable (latest release) in German
+# This file is distributed under the same license as the Plugins - WooCommerce Blocks - Stable (latest release) package.
+msgid ""
+msgstr ""
+"PO-Revision-Date: 2022-05-22 10:58:25+0000\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=n != 1;\n"
+"X-Generator: GlotPress/4.0.0-alpha.1\n"
+"Language: de\n"
+"Project-Id-Version: Plugins - WooCommerce Blocks - Stable (latest release)\n"
+
+#: assets/js/blocks/featured-product/block.json
+#: build/featured-product/block.json
+msgctxt "block description"
+msgid "Visually highlight a product or variation and encourage prompt action."
+msgstr "Ein Produkt oder eine Variante visuell hervorheben und zum sofortigen Handeln auffordern."
+
+#: assets/js/blocks/featured-product/block.json
+#: build/featured-product/block.json
+msgctxt "block title"
+msgid "Featured Product"
+msgstr "Hervorgehobenes Produkt"
+
+[...]
+```
+
+See also https://developer.wordpress.org/plugins/internationalization/localization/#po-portable-object-files.
+
+### MO (Machine Object) file
+
+MO stands for `Machine Object`, contains both the original and the translated strings and is based on the PO file. As the POT and the PO files, the MO file can be created manually, but this is not necessary for the WooCommerce Blocks plugin.
+
+The MO file is only machine-readable and named `woo-gutenberg-products-block-{LANGUAGE-CODE}.mo`, e.g. `woo-gutenberg-products-block-de_DE.mo` It can be found in the `/wp-content/languages/plugin` folder. As the MO file is machine-readable only, it cannot be viewed and the MO file only handeles translations within PHP files.
```

Comment:
```suggestion
The MO file is only machine-readable and named `woo-gutenberg-products-block-{LANGUAGE-CODE}.mo`, e.g. `woo-gutenberg-products-block-de_DE.mo` It can be found in the `/wp-content/languages/plugin` folder. As the MO file is machine-readable only, it cannot be viewed and the MO file only handles translations within PHP files.
```